lib/Test2/Tools/Spec.pm

```diff
@@ -524,6 +568,50 @@ Same as:
 
 =back
 
+=head2 CUSTOM ATTRIBUTE DEFAULTS
+
+Sometimes you want to apply default attributes to all C<tests()> or C<case()>
+blocks. This can be done, and is lexical to your describe or package root!
+
+    use Test2::Bundle::Extended;
+    use Test2::Tools::Spec ':ALL';
+
+    # All 'tests' blocks after this declaration will have C<<iso => 1>> by default
+    spec_defaults tests => (iso => 1);
+
+    tests foo => sub { ... }; # isolated
+
+    tests foo, {iso => 0}, sub { ... }; # Not isolated
+
+    spec_Defaults tests => (iso => 0); # Turn it off again
+
+Defaults are inherited by nested describe blocks. You can also override the
+defaults for the scope of the describe:
+
+    spec_defaults tests => (iso => 1);
+
+    describe foo => sub {
+        spec_defaults tests => (async => 1); # Scoped to this describe and any child describes
+
+        tests bar => sub { ... }; # both iso and async
+    };
+
+    tests baz => sub { ... }; # Just iso, no async.
+
+You can apply defaults to any type of blocks:
+
+    spec_defaults case => (iso => 1); # All cases are 'iso';
+
+Defaults are not inherited when a builder is constructed instead of added to
```

Comment:
A little hard to understand. How about "Defaults are not inherited from the current build when a builder is constructed."